.php_cs

```diff
@@ -21,36 +21,19 @@ EOF;
 return PhpCsFixer\Config::create()
     ->setRiskyAllowed(true)
     ->setRules([
-        'array_indentation' => true,
-        'array_syntax' => ['syntax' => 'short'],
+        '@Symfony' => true,
         'binary_operator_spaces' => ['align_double_arrow' => true, 'align_equals' => true],
+        'array_syntax' => array('syntax' => 'short'),
         'blank_line_after_namespace' => true,
-        'blank_line_after_opening_tag' => true,
-        'declare_strict_types' => true,
-        'header_comment' => ['header' => $header],
-        'indentation_type' => true,
-        'linebreak_after_opening_tag' => true,
+        'header_comment' => array('header' => $header),
         'lowercase_constants' => true,
         'lowercase_keywords' => true,
         'method_separation' => true,
-        'native_function_invocation' => ['include' => ['@compiler_optimized']],
-        'no_alias_functions' => true,
         'no_closing_tag' => true,
         'no_extra_consecutive_blank_lines' => [
```

Comment:
Why overriding Symfony rules?

Reply:
As far as I know `no_extra_consecutive_blank_lines` is not part of `@Symfony`...

Reply:
1. `no_extra_consecutive_blank_lines` is deprecated, replaced by `no_extra_blank_lines `
2. see https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/2.15/src/RuleSet.php#L90

We should probably fix it in an another PR